eng/Versions.props

```diff
@@ -136,7 +136,7 @@
     <MicrosoftVisualStudioTextManagerInterop120Version>12.0.30110</MicrosoftVisualStudioTextManagerInterop120Version>
     <MicrosoftVisualStudioTextManagerInterop121DesignTimeVersion>12.1.30328</MicrosoftVisualStudioTextManagerInterop121DesignTimeVersion>
     <MicrosoftVisualStudioThreadingAnalyzersVersion>15.8.168</MicrosoftVisualStudioThreadingAnalyzersVersion>
-    <MicrosoftVisualStudioThreadingVersion>15.8.99-rc</MicrosoftVisualStudioThreadingVersion>
+    <MicrosoftVisualStudioThreadingVersion>15.8.168</MicrosoftVisualStudioThreadingVersion>
```

Comment:
So long as you're upgrading, why not upgrade to 16.0?

Reply:
The branch @heejaechang is targeting still must be runnable in 15.* to support open source contributors. I assume 16.0 is only available in 16.0?

Reply:
Yep, we should be able to update to 16.0 after 16.0 RTM.

Reply:
I dont know, usually, we use versions in sync with other dependency. the new StreamJonRpc I am using is depends on this specific version and there is no other dependency that use higher version than this. so I am following same pattern.